generators/Exercises/ComplexNumbers.cs

```diff
@@ -20,38 +20,30 @@ protected override void UpdateCanonicalData(CanonicalData canonicalData)
             {
                 // Process expected
                 if (IsComplexNumber(canonicalDataCase.Expected))
-                {
```

Comment:
Haha, true. I tend to not bother with these kind of changes since it does boil down to preferences, but sometimes you just cant resist. I love braces!

Reply:
@robkeim @jpreese Yeah, there is some stylistic difference between us ;) Perhaps we should start a discussion at #398.

Reply:
@ErikSchierboom ok sounds good!